src/main/scala/com.snowplowanalytics.snowplow/scalatracker/Tracker.scala

```diff
@@ -23,10 +23,13 @@ import org.json4s._
 import org.json4s.JsonDSL._
 
 import com.snowplowanalytics.iglu.core.{ SelfDescribingData, SchemaKey, SchemaVer }
+import com.snowplowanalytics.snowplow.scalatracker.utils.ErrorTracking._
 import com.snowplowanalytics.iglu.core.json4s.implicits._
 
 import emitters.TEmitter
 
+import scala.util.control.NonFatal
+
```

Comment:
is this import used anywhere?

By the way I agree that if we're providing an API with `Throwable` we should take care of all throwables and not no-op on them. We should either change the API or handle all throwables without feeling responsible for other people's bad practices.

Reply:
I removed the import.